packages/ui/src/components/textarea.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+import { cn } from '../utils/cn'
+
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(({ className, ...props }, ref) => {
+  return (
+    <textarea
+      className={cn(
+        'border-input placeholder:text-muted-foreground focus-visible:ring-ring flex min-h-[60px] w-full rounded-md border bg-transparent px-3 py-2 text-sm shadow-sm focus-visible:outline-none focus-visible:ring-1 disabled:cursor-not-allowed disabled:opacity-50',
```

Comment:
Let’s make sure to use the new color variables, which can be found in the [tailwind-config](https://github.com/harness/canary/blob/main/packages/ui/tailwind.ts#L90) and [styles](https://github.com/harness/canary/blob/main/packages/ui/src/styles.css#L99):

For borders: `borders`
For text: `foreground`
For background: `background`

Therefore, it should be:
```suggestion
        'border-borders-2 placeholder:text-foreground-5 flex min-h-[60px] w-full rounded border bg-transparent px-3 py-2 text-14 disabled:cursor-not-allowed disabled:opacity-50',
```


- `focus-visible` is currently set globally in the [styles](https://github.com/harness/canary/blob/main/packages/ui/src/styles.css#L1519), so there’s no need to apply it manually. This was done to ensure consistency across the entire project.
- `shadow-sm` – shadows are not used on input components in the updated design.
- `text-sm` => `text-14` – We use this approach to think in terms of `px` for easier understanding of text sizes. A value like sm doesn’t convey much, whereas 14 gives a clearer idea of the actual size. You can find more details about this in the [tailwind.config](https://github.com/harness/canary/blob/main/packages/ui/tailwind.ts#L248)

Reply:
no problem